examples/todos/package.json

```diff
@@ -6,6 +6,7 @@
   "license": "MIT",
   "dependencies": {
     "@elastic/synthetics": "*",
+    "playwright-chromium": "^1.9.1",
```

Comment:
can you remove this one. Not required.

Reply:
i didn't add this, it got added auto. idk how.

Reply:
done